documentation/modules/exploit/multi/http/open_web_analytics_rce.md

```diff
@@ -0,0 +1,64 @@
+## Vulnerable Application
```

Comment:
A comment for this entire file.

Per the markdown spec, string literals that do not span multiple lines should be enclosed in single back ticks ``` ` ```.

Literals that span multiple lines should be enclosed in triple back ticks that is 3 and not 4 as you're using here.